### PR TITLE
Add XPLA chain

### PR DESCRIPTION
**Pull request by Moonyongjung**
Add XPLA chain to chain-registry

### CODE REVIEW

File: xpla/chain.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "xpla",
+  "status": "live",
+  "website": "https://xpla.io",
+  "network_type": "mainnet",
+  "pretty_name": "XPLA",
+  "chain_id": "dimension_37-1",
+  "bech32_prefix": "xpla",
```
Comment (JeremyParish69) [inline xpla/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: xpla/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          "exponent": 18
+        }
+      ],
+      "base": "axpla",
+      "name": "Xpla",
+      "display": "xpla",
+      "symbol": "XPLA",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/xpla/images/xpla.svg",
+	      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/xpla/images/xpla.png"
```
Comment (JeremyParish69) [inline xpla/assetlist.json:23]: Please either change these references to site the upper case file names (e.g., .../images/xpla.png -> .../images/XPLA.png), or else make the actual image names lower case (preferred) (e.g., XPLA.png -> xpla.png).

Comment (Moonyongjung) [reply xpla/assetlist.json:23]: Change file name of images

File: xpla/chain.json (hunk 1 above)
Comment (Moonyongjung) [reply xpla/chain.json:9]: I register the xpla prefix on SLIP-0173

**Pull request merged**